pkg/bootstrap/docker/up.go

```diff
@@ -511,6 +518,38 @@ func (c *ClientStartConfig) EnsureHostDirectories(io.Writer) error {
 	return c.HostHelper().EnsureVolumeShare()
 }
 
+// EnsureDefaultRedirectURIs merges a default URL to an auth client's RedirectURIs array
+func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
+	oc, _, err := c.Clients()
+	if err != nil {
+		return nil
+	}
+
+	webConsoleOAuth, err := oc.OAuthClients().Get(defaultRedirectClient)
+	if err != nil {
+		return err
+	}
+
+	// ensure the default redirect URI is not already present
+	redirects := sets.NewString(webConsoleOAuth.RedirectURIs...)
+	if redirects.Has(defaultRedirectURI) {
+		return nil
+	}
+
+	webConsoleOAuth.RedirectURIs = append(webConsoleOAuth.RedirectURIs, defaultRedirectURI)
+
+	_, err = oc.OAuthClients().Update(webConsoleOAuth)
+	if err != nil {
+		// announce error without interrupting remaining tasks
+		suggestedCmd := fmt.Sprintf("oc patch %s -p '{%q:[%q]}'", "oauthclient/openshift-web-console", "redirectURIs", defaultRedirectURI)
+		errMsg := fmt.Sprintf("Unable to set %q as a default redirect URL for the web console.\nTo manually add it, run %q\n", defaultRedirectURI, suggestedCmd)
```

Comment:
"unable to add development redirect URI to the openshift-web-console OAuthClient..."